web/themes/joinup/joinup_theme.theme

```diff
@@ -130,6 +130,13 @@ function joinup_theme_preprocess_page_title(&$variables) {
   }
   $route_name = $route_match->getRouteName();
 
+  $variables['visually_hidden_class'] = '';
+
+  // Early bail out.
+  if (empty($route_name)) {
```

Comment:
This can be even before `$variables['visually_hidden_class'] = '';`, but it's ok.